### PR TITLE
Update Configure page design

### PR DESCRIPTION
**Pull request by JReinhold**
This PR updates the style, layout and assets of the Configure MDX doc.

See the changes here: https://update-configure-page-styles--646cb5324472138d1d7d6720.chromatic.com/?path=/docs/configure-your-project--docs

<details>
  <summary>📦 Published PR as canary version: <code>1.0.4-canary.74.85c1178.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/addon-onboarding@1.0.4-canary.74.85c1178.0
  # or 
  yarn add @storybook/addon-onboarding@1.0.4-canary.74.85c1178.0
  ```
</details>

### CODE REVIEW

**Review by cdedreuille — APPROVED** [review]

File: src/components/SyntaxHighlighter/SyntaxHighlighter.stories.tsx [hunk 1]
```diff
@@ -88,40 +85,4 @@ export const Default: Story = {
     width: 480,
     filename: "Button.stories.tsx",
   },
-  play: async ({ canvasElement }) => {
```
Comment (JReinhold) [inline src/components/SyntaxHighlighter/SyntaxHighlighter.stories.tsx:91]: Why did you remove this @cdedreuille ?

**Comment by valentinpalkovic** [comment]
The bottom sections seems to be broken on smaller devices
image

**Review by JReinhold — COMMENTED** [review]
@cdedreuille 

1. In smaller viewports (but still bigger than mobile) the logos cover the text:
image

2. In Chrome, the rotation of the logos produces noticable jagged edges

Chrome:
image

Firefox and Safari:
image


I tried a bunch of things from [this SO question](https://stackoverflow.com/questions/6492027/css-transform-jagged-edges-in-chrome), but the only thing that removed them was the combination of:

```css
.sb-addon-img img {
  box-shadow: 0 0 1px rgba(255, 255, 255, 0);
  backface-visibility: hidden;
}
```

File: src/components/SyntaxHighlighter/SyntaxHighlighter.stories.tsx (hunk 1 above)
Comment (cdedreuille) [reply src/components/SyntaxHighlighter/SyntaxHighlighter.stories.tsx:91]: We talked about it with Valentin and this play function was very flaky and created unexpected behaviours on Chromatic. We decided to remove it as it was not really adding any value.

Comment (JReinhold) [reply src/components/SyntaxHighlighter/SyntaxHighlighter.stories.tsx:91]: Deal!

**Comment by cdedreuille** [comment]
> The bottom sections seems to be broken on smaller devices image

This is now fixed @valentinpalkovic

**Review by valentinpalkovic — APPROVED** [review]
